app/views/claims/_school_search.html.erb

```diff
@@ -33,7 +33,7 @@
         Enter the school name. Use at least four characters.
       </span>
 
-      <% if school_id_param == :claim_school_id && !params.has_key?(:additional_school) %>
+      <% if current_policy_routing_name == "student-loans" && school_id_param == :claim_school_id && !params.has_key?(:additional_school) %>
```

Comment:
Do we want to have this added conditional here? I would have thought the journey for multiple schools would be the same for Maths and Physics as for Student Loans?

Reply:
The policy for Maths and Physics is that you are currently employed at an eligible school, so we don't need to ask for multiple schools, you have to still be working at one to claim.

The school search partial is re-used across the policies so we need this conditional to prevent mis-informing maths and physics users.

As an aside, I do think we could look to making the school search partials a bit neater, but I am not sure it should be part of this work?

Reply:
@mec and I were talking about this a bit earlier. I noted that if we called this field `current_school` instead of `claim_school`, then the flow would indeed be the same as the Student Loans flow, and we wouldn't need the conditional.

But then, I wonder whether it even makes sense to prefix the school in Maths and Physics. There is only one school, so why not just call it `school`, I wonder?